### ISSUE TITLE
Flat nodes not working or the bat?

### ISSUE THREAD

**Issue by ElbowedGolem** [brumenn/FlatNodes#14]
Does Flat nodes require anything other than just enabling it in the plugins section?  i have enabled it and notice no difference, I'm on ue5.3 thanks

**Comment by brumenn** [comment]
Did you restart the engine ?

**Comment by ElbowedGolem** [comment]
> Did you restart the engine ?

Hi, yes I did. Hence I thought maybe you have to set something other than just enabling it. I looked in the plugins in the engine folder inside ue editor and the only thing that was there was a signal material. I didn't bother with the c++ folder as I know extremely little about it.

**Comment by ElbowedGolem** [comment]
> > Did you restart the engine ?
> 
> Hi, yes I did. Hence I thought maybe you have to set something other than just enabling it. I looked in the plugins in the engine folder inside ue editor and the only thing that was there was a signal material. I didn't bother with the c++ folder as I know extremely little about it.

I installed this from the ue marketplace

**Comment by brumenn** [comment]
No, there's nothing else to do normally, and there really aren't many files, even in C++..
So I don't really know where the problem could come from honestly.

**Comment by ElbowedGolem** [comment]
> No, there's nothing else to do normally, and there really aren't many files, even in C++.. So I don't really know where the problem could come from honestly.

Ahh okay, we'll thanks anyway, I'll try completely uninstalling it and re install it again and see if it's fixed
